browser/css/jsdialogs.css

```diff
@@ -1535,28 +1465,3 @@ div [id^='TemplateDialog'] {
 #CharacterPropertiesDialog {
 	min-width: 656px;
 }
-
-/* Keyboard input keys */
-kbd,
-.kbd {
-	background-color: var(--color-background-dark);
-	border-radius: var(--border-radius);
-	border: 1px solid var(--color-border-dark);
-	color: var(--color-text-dark);
-	display: inline-block;
-	font-family: monospace;
-	font-size: .85em;
-	line-height: 1;
-	padding: 4px;
-	white-space: nowrap;
-}
-
-.kbd--plus {
-	color: var(--color-text-dark);
-	padding: 0 4px;
-	font-size: 0.85em;
-}
-
-.warn-copy-paste {
-	width: 100%;
-}
```

Comment:
Please fix this PR or close and create another one. There are here many unrelated code changes, it seems, as a result from solving conflicts when rebasing it locally

Reply:
Noted, I am going to create a new PR as all these changes are caused after I pulled the repo